<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>544</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font: 13px &quot;맑은 고딕&quot;;color: rgb(63, 63, 63);
background-color: None;
}


/*** 버튼 ***/
QPushButton {background-color: transparent;border-radius: 0px;border-style: solid;border-width: 1px;border-color: transparent}

QPushButton:hover {background-color: rgb(230, 230, 230);}

QPushButton:pressed {background-color: rgb(205, 205, 205);}

QPushButton:focus {
border-color: rgb(200,200,200);
}</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {font: 15px; color: rgb(127, 127, 127);}</string>
   </property>
   <property name="text">
    <string>알림</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/**프레임**/
QFrame{
background-color: white;
border-radius: 0px;
border-style:solid;
border-width:0px;
/*border-color: rgb(127, 127, 127);*/
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="yes_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="icon">
     <iconset resource="../QRC/img.qrc">
      <normaloff>:/__resource__/image/unit_yes.png</normaloff>:/__resource__/image/unit_yes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="no_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="icon">
     <iconset resource="../QRC/img.qrc">
      <normaloff>:/__resource__/image/unit_no.png</normaloff>:/__resource__/image/unit_no.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>29</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:13px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13px;&quot;&gt;텍스트 이곳에 표시&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <zorder>text</zorder>
   <zorder>yes_btn</zorder>
   <zorder>no_btn</zorder>
  </widget>
  <widget class="QPushButton" name="msg_close_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: transparent;border-radius: 0px;border-style: solid;border-width: 3px;border-color: transparent;height: 24px;width: 80px;
margin-left: 1px;
margin-right: 1px;
margin-top: 1px;
image: url(:/__resource__/image/unit_close.png);}

QPushButton:hover {background-color: rgb(232, 17, 35);
image: url(:/__resource__/image/unit_close_pressed.png);}
QPushButton:pressed {background-color: rgb(241, 112, 122);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>title</zorder>
  <zorder>msg_close_btn</zorder>
 </widget>
 <resources>
  <include location="../../__resource__/resource.qrc"/>
 </resources>
 <connections/>
</ui>
